templates/zerver/app/delete_message.html

```diff
@@ -17,7 +17,7 @@ <h3 id="delete_message_modal_label">
     </div>
     <div class="modal-footer">
         <button class="button rounded" data-dismiss="modal">{{ _("Cancel") }}</button>
-        <button class="button rounded btn-danger" id="do_delete_message_button">{{ _("Yes, delete this message") }}</button>
+        <button class="button rounded btn-danger" id="do_delete_message_button">{{ _("Confirm") }}</button>
```

Comment:
![image](https://user-images.githubusercontent.com/59444243/114265944-376ddb00-9a11-11eb-9ed7-0cbee03bd265.png)
Oops! Sorry, I miss this fix to add in my previous PR #17947. So I have covered this fix in my new commit here.